pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -176,8 +176,8 @@ type SyncStatus struct {
 	// +listType=atomic
 	Message []string `json:"message"`
 
-	// The repository hash when the last sync ran
-	Hash string `json:"hash,omitempty"`
+	// The repository ref when the last successful sync ran
+	LastRef string `json:"last_ref,omitempty"`
```

Comment:
```suggestion
	LastRef string `json:"lastRef,omitempty"`
```
we are using camel case elsewhere... but not strong feelings here :)

Reply:
Yeah, let's use camelCase, because that also gets propagated to FE.